main.py

```diff
@@ -20,13 +24,16 @@ def next_day(dt):
 
 
 class BinanceMDS:
-    def __init__(self, ws_endpoint, symbol, folder='', loop=asyncio.new_event_loop()):
+    def __init__(self, ws_endpoint: str, symbol: str, directory: str = '',
+                 loop: Optional[asyncio.AbstractEventLoop] = None):
```

Comment:
For what? It makes code harder

Reply:
https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

I’m not sure if this is the case for event loop and function that creates it. I just thought that it looks a little strange - calling function to get default parameter